librz/include/rz_analysis.h

```diff
@@ -728,17 +730,19 @@ typedef struct dwarf_variable_t {
 typedef enum {
 	RZ_ANALYSIS_VAR_ORIGIN_NONE = 0, ///< Variable was created from rizin
 	RZ_ANALYSIS_VAR_ORIGIN_DWARF, ///< Variable was created from DWARF information
+	/* End enum */
+	RZ_ANALYSIS_VAR_ORIGIN_END  ///< Number of RzAnalysisVarOriginKind enums
 } RzAnalysisVarOriginKind;
 
-static const char *RzAnalysisVarKind_strings[] = {
-	[RZ_ANALYSIS_VAR_KIND_INVALID] = "invalid",
-	[RZ_ANALYSIS_VAR_KIND_FORMAL_PARAMETER] = "formal_parameter",
-	[RZ_ANALYSIS_VAR_KIND_VARIABLE] = "variable",
+static const char *RzAnalysisVarKind_strings[RZ_ANALYSIS_VAR_KIND_END] = {
+	"invalid", /* RZ_ANALYSIS_VAR_KIND_INVALID */
+	"formal_parameter", /* RZ_ANALYSIS_VAR_KIND_FORMAL_PARAMETER */
+	"variable" /* RZ_ANALYSIS_VAR_KIND_VARIABLE */
 };
 
 static const char *RzAnalysisVarOriginKind_strings[] = {
```

Comment:
Be consistent and use size here too, or do not use it anywhere

Reply:
Thanks, missed it